src/main.js

```diff
@@ -70,52 +70,52 @@ const routes = [
   // Tutorial 0001
   { path: '/data-structures', component: Landing, props: { tutorialId: '0001' } },
   { path: '/data-structures/resources', component: ResourcesLesson, props: { tutorialId: '0001' } },
-  { path: '/data-structures/01', component: T0001L01 },
-  { path: '/data-structures/02', component: T0001L02 },
-  { path: '/data-structures/03', component: T0001L03 },
-  { path: '/data-structures/04', component: T0001L04 },
-  { path: '/data-structures/05', component: T0001L05 },
+  { path: '/data-structures/01', component: T0001L01, props: { tutorialId: '0001' } },
```

Comment:
These changes are to a part of the code created by the tutorial author, so we'll have to update our documentation. Can you please update the Developing Tutorials doc accordingly as part of this PR?

Reply:
done